src/Codegen/Constraints/ObjectBuilder.php

```diff
@@ -525,7 +525,7 @@ private function codegenType(
           ? $additional_properties
           : true;
 
-      $members = [];
+      $members = vec[];
```

Comment:
Members is only used in append and ...$members expressions, so the type difference can not be observed.